magpie/setup/magpie-setup-project-hadoop

```diff
@@ -1006,6 +1006,13 @@ then
         yarnauxmapreduceshuffle="yarn.nodemanager.aux-services.mapreduce_shuffle.class"
     fi
 
+    if [ "${YARN_VMEM_CHECK}" == "false" ]
+    then
+        yarnvmemcheck='false'
+    else
+        yarnvmemcheck='true'
+    fi
+
```

Comment:
I'm not familiar with this environment variable.  Is this coming from Spark or Zeppelin?  Or is this from your code?  Should it be a new config in Magpie?